cpp/daal/include/algorithms/decision_forest/decision_forest_training_parameter.h

```diff
@@ -176,6 +176,11 @@ class DAAL_EXPORT Parameter
     double minImpurityDecreaseInSplitNode; /*!< A node will be split if this split induces a decrease of the impurity
                                                   greater than or equal to the value, non-negative. Default is 0.0 */
     size_t maxLeafNodes;                   /*!< Maximum number of leaf node. Default is 0 (unlimited) */
+    size_t maxBins;                        /*!< Used with 'hist' split finding method only.
```

Comment:
1. need to check dynamic BC. 
2. you need to get approval for expanding the interfaces (for both interfaces: old and new).

Reply:
Checked that dynamic BC is not affected with these changes.